mockApis/communityApi.ts

```diff
@@ -7,7 +7,7 @@ export default class CommunityApiMocks {
     return this.wiremock.stubFor({
       request: {
         method: 'GET',
-        urlPattern: `/community-api/secure/offenders/crn/${crn}`,
+        urlPattern: `/community-api/secure/offenders/crn/${crn}/all`,
```

Comment:
Have just realised this response might be massive - am going to make this an additional endpoint so we're not calling it on every single page for the SU banner.

Reply:
Have reworked this.